classpath/java/lang/invoke/SerializedLambda.java

```diff
@@ -0,0 +1,48 @@
+/* Copyright (c) 2008-2016, Avian Contributors
+
+   Permission to use, copy, modify, and/or distribute this software
+   for any purpose with or without fee is hereby granted, provided
+   that the above copyright notice and this permission notice appear
+   in all copies.
+
+   There is NO WARRANTY for this software.  See license.txt for
+   details. */
+
+package java.lang.invoke;
+
+public class SerializedLambda implements java.io.Serializable {
```

Comment:
Not really a decision we can make as downstream spec implementors, but it seems to me that serializing lambdas is just asking for some sort of security vulnerability.